contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -117,3 +131,55 @@ func execTxsAddPackages(
 
 	return nil
 }
+
+func signTxs(txs []gnoland.TxWithMetadata, signer *gnoclient.SignerFromKeybase) error {
+	for index, tx := range txs {
+		signBytes, err := tx.Tx.GetSignBytes(signer.ChainID, 0, 0)
```

Comment:
We should add a comment here to note that the account sequence and account number are not checked as part of the genesis tx sig validation

Reply:
Added a comment on c967c2f43104f565d44d7dcf789cfb7cdf1798e5